README.md

```diff
@@ -5765,12 +5765,42 @@ Interface to send syslog data from, e.g. "management".  Valid value is a string.
 
 #### Parameters
 
-##### `name`

```

Comment:
Please add a caveats table and indicate that these properties are being added in version `1.10.0` of the module.

Here is an example:

https://github.com/cisco/cisco-network-puppet-module/blob/86132da1bbf4c4b34199675fb5e286100e68f0df/README.md#caveats-5

Also please add a link under the Caveats column in the support matrix.

https://github.com/cisco/cisco-network-puppet-module#resource-platform-support-matrix

Reply:
Thanks Mike, I have added the Caveat and link